packages/a/arduino-ide/files/arduino.ide.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>arduino-ide.desktop</id>
```

Comment:
Assuming this is Arduino IDE V2, `flathub` uses the id `cc.arduino.IDE2`. We probably want to use the same thing to avoid duplicate entries in software centers. https://github.com/flathub/cc.arduino.IDE2/blob/master/cc.arduino.IDE2.appdata.xml